puppet-module-repose.spec

```diff
@@ -30,6 +30,8 @@ cp -pr * %{buildroot}%{module_dir}/
 %{module_dir}
 
 %changelog
+* Tue Jan 17 2017 Josh Bell <josh.bell@rackspace.com> - 1.8.3-1
```

Comment:
I think I would rather leave this file until we have the metadata.json file there. Updated the version in the changelog.